packages/jest-each/src/__tests__/array.test.js

```diff
@@ -0,0 +1,200 @@
+import each from '../';
```

Comment:
it would be awesome to have an integration test with snapshots of the output for it eventually!